src/content/getting-started/settings.mdx

```diff
@@ -3,20 +3,155 @@ title: Settings
 description: Learn how to interact with WordPress Options (settings) using WPGraphQL
 ---
 
-WordPress has a `register_settings()` API that allows developers to register settings for their
-site.
+## Core Settings
 
-WPGraphQL respects this API and exposes registered settings in the WPGraphQL Schema for querying
-and mutating.
+WPGraphQL respects the WP core Settings API and exposes all core settings in the WPGraphQL Schema for querying and mutating. Any settings that are registered through `register_settings()` can be added to the schema.
+
+import GraphiQL from '../../components/GraphiQL'
+
+## Registering Custom Settings
+
+WordPress has a `register_settings()` API that allows developers to register settings for their site. Check [here](https://developer.wordpress.org/reference/functions/register_setting/) for more information on how these settings are registered by group and name. In WordPress you could register a custom setting like so:
+
+```PHP
+/**
+ * Registers a text field setting for Wordpress 4.7 and higher.
+ **/
+function register_my_setting() {
+    $args = array(
+            'type' => 'string',
+            'sanitize_callback' => 'sanitize_text_field',
+            'default' => NULL,
+            'show_in_graphql' = true,
+            );
+    register_setting( 'my_options_group', 'my_option_name', $args );
+}
+add_action( 'admin_init', 'register_my_setting' );
+```
+
+Since we set the `show_in_graphql` parameter to true the setting will now appear in the GraphQL schema under it's group name. If the setting was registered without one it will appear under generalSettings.
 
 ## Queries
 
-`@todo: Provide examples`
+### All Settings
 
-## Mutations
```

Comment:
we should still document mutations here so folks have an idea how to do them, even if the execution in the `<GraphiQL>` component in the docs will not work without auth. Can even just document examples in a code block instead of `<GraphiQL>`